.drone.yml

```diff
@@ -265,6 +268,7 @@ steps:
     build_args:
     - ARCH=arm64
     - "VERSION=${DRONE_TAG}"
+    - SYSTEM_CHART_DEFAULT_BRANCH=master
```

Comment:
drop this. want to use the dockerfile. ultimtately, we can break RC building out from real releases if we want